### ISSUE TITLE
[Suggestion] Copy/Paste settings in Gravity/Shipyard blocks

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#2745]
This function would make life so much easier when dealing with blocks that are like these and require you to play around with the numbers to get the desired area. Copy and paste for settings is probably more useful for shipyards since most people seem to lean towards those for carrying ships around. I don't think this is a super important thing to get in the game right away but I do think it would be a nice quality of life sort of thing.

**Comment by tsunamayo** [comment]
Yes I can understand that. I think more important would be a way to stretch the bounding box manually, as inputting value is quite tedious as you dont really see where it goes... Cheers

**Comment by Xerma** [comment]
That would be far better and much easier to use
